src/common/Collision/Management/VMapManager2.cpp

```diff
@@ -146,9 +142,7 @@ namespace VMAP
             Vector3 pos1 = convertPositionToInternalRep(x1, y1, z1);
             Vector3 pos2 = convertPositionToInternalRep(x2, y2, z2);
             if (pos1 != pos2)
-            {
                 return instanceTree->second->isInLineOfSight(pos1, pos2);
-            }
         }
```

Comment:
this is not good, we want to keep brackets even for 1-line if/for blocks, this is in order to prevent human mistakes